src/main.js

```diff
@@ -80,7 +80,7 @@ const __bigFT = (function (){
 
 	const newsTicker = $('.ticker').ticker();
 	const mediaHolder = document.getElementsByClassName('main-stories')[0];
-	const clocks = document.querySelectorAll('[data-tz]');
+	let clocks = undefined;
```

Comment:
The amount of clocks can change, so this needs to be assigned after all required clocks have been generated and added to the DOM